ui/templates/base.html

```diff
@@ -7,6 +7,10 @@
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
     </script>
+    {% if style_public_src %}
+    <script type="text/javascript" src="{{ style_public_src }}"></script>
+    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
```

Comment:
well, aren't `<script>` tags executed in the order of their declaration, regardless of how long the request takes? we aren't setting the `async` or `defer` attributes.